modules/memberships/class-jetpack-memberships.php

```diff
@@ -0,0 +1,283 @@
+<?php
+/**
+ * Class Jetpack_Memberships
+ * This class represents the Memberships functionality.
+ */
+class Jetpack_Memberships {
+	/**
+	 * CSS class prefix to use in the styling.
+	 *
+	 * @var string
+	 */
+	static public $css_classname_prefix = 'jetpack-memberships';
+	/**
+	 * Increase this number each time there's a change in CSS or JS to bust cache.
+	 *
+	 * @var string
+	 */
+	static private $version = JETPACK__VERSION;
+	/**
+	 * Our CPT type for the product (plan).
+	 *
+	 * @var string
+	 */
+	static public $post_type_plan = 'jp_mem_plan';
+	/**
+	 * @var string Option that will store currently set up account (Stripe etc) id for memberships
+	 */
+	static public $connected_account_id_option_name = 'jetpack-memberships-connected-account-id';
+	/**
+	 * Button block type to use.
+	 *
+	 * @var string
+	 */
+	static private $button_block_name = 'membership-button';
+	/**
+	 * Classic singleton pattern
+	 *
+	 * @var Jetpack_Memberships
+	 */
+	private static $instance;
+
+	/**
+	 * Jetpack_Memberships constructor.
+	 */
+	private function __construct() {}
+
+	/**
+	 * The actual constructor initializing the object.
+	 *
+	 * @return Jetpack_Memberships
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+			self::$instance->register_init_hook();
+		}
+
+		return self::$instance;
+	}
+	/**
+	 * Get the map that defines the shape of CPT post. keys are names of fields and
+	 * 'meta' is the name of actual WP post meta field that corresponds.
+	 *
+	 * @return array
+	 */
+	private static function get_plan_property_mapping() {
+		$meta_prefix = 'mem_';
```

Comment:
Would `memberships_` be more clear and meaningful? Now folks need to look this code up to understand what this is when they see the option in database. Do we need to prefix these with `jetpack_`?